Python/ceval.c

```diff
@@ -442,7 +442,8 @@ _Py_InitializeRecursionLimits(PyThreadState *tstate)
     _tstate->c_stack_soft_limit = _tstate->c_stack_hard_limit + PYOS_STACK_MARGIN_BYTES;
 #else
     uintptr_t here_addr = _Py_get_machine_stack_pointer();
-#  if defined(HAVE_PTHREAD_GETATTR_NP) && !defined(_AIX) && !defined(__NetBSD__)
+#  if defined(HAVE_PTHREAD_GETATTR_NP) && !defined(_AIX) && \
+        !defined(__NetBSD__) && defined(__GLIBC__)
```

Comment:
I don't think that this call is correct. It disables the call on FreeBSD.

Maybe check for "glibc or not Linux"?

Reply:
That would fix it for alpine, but not for emscripten or wasi (but I don't know what they actually need).  I'd *rather* see it as if glibc or freebsd, and get rid of the rest, but glibc or not linux would work for my immediate concern.

Reply:
Well, let's start with fixing Alpine Linux.